admin_pages/transactions/templates/txn_admin_details_main_meta_box_txn_details.template.php

```diff
@@ -134,16 +134,16 @@ class="hide-the-displayed hidden smaller-text"
                                 : '{}';
                             $escaped_pay_id = esc_attr($PAY_ID);
                             ?>
-                            <tr id="txn-admin-payment-tr-<?php echo $escaped_pay_id; ?>">
+                            <tr id="txn-admin-payment-tr-<?php echo esc_attr($escaped_pay_id); ?>">
```

Comment:
OOF... this is essentially being triple sanitized now just within this template...

line 131:
```php
$PAY_ID = absint($PAY_ID);
```

line 135:
```php
$escaped_pay_id = esc_attr($PAY_ID);
```

and now on lines 137, 139, 143, 146... etc
```php
echo esc_attr($escaped_pay_id);
```

let's remove the initial sanitization and then replace the usage within the HTML with:
```php
<tr id="txn-admin-payment-tr-<?php echo absint($PAY_ID); ?>">
```

i'm obviously responsible for the initial double escaping which then hid what was going on from you.

but again, let's just use `absint()` for known integer values